hooks/useAlbums.ts

```
@@ -0,0 +1,13 @@
+import { getAlbums } from "@/services/albumService";
+import type { AppDispatch, RootState } from "@/store/store";
+import { useDispatch, useSelector } from "react-redux";
+
+export const useAlbums = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const albums = useSelector((state: RootState) => state.albums.albums);
+  const status = useSelector((state: RootState) => state.albums.status);
+
+  const getAlbumsAction = (data: any) => dispatch(getAlbums(data.uid));
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)